cpu/esp8266/vendor/esp-idf/Kconfig

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ESP_IDF
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_ESP8266
+
+rsource "esp8266/source/Kconfig"
+rsource "heap/src/Kconfig"
+rsource "nvs_flash/Kconfig"
```

Module is not compiled in `nvs_flash` but in `nvs_flash/src`. I guess the `Kconfig` has to be in that directory to use the right makefile. 
```suggestion
rsource "nvs_flash/src/Kconfig"
```

---

Hmm now that you point this out I'm confused. Why is this makefile there, with the module name?

https://github.com/RIOT-OS/RIOT/blob/master/cpu/esp8266/vendor/esp-idf/nvs_flash/Makefile

---

It's obviously a leftover. Compilation works without this Makefile. You could remove it with this PR.

---

Will do.

---

Makefile removed here 4b09f491ca921f35503d493d99085f3814c48ff0.